<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="silius" errors="0" failures="66" skipped="13" tests="486" time="266.493" timestamp="2025-06-03T07:39:25.793885+00:00" hostname="pkrvmf6wy0o8zjz"><testcase classname="" name="tests.single.gas.test_pre_verification_gas_calculation" time="0.000"><skipped message="collection skipped">('/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/gas/test_pre_verification_gas_calculation.py', 15, 'Skipped: Slow and requires fixing')</skipped></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[only_priority_fee_bump]" time="0.252" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[only_max_fee_bump]" time="0.176" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[with_same_fee]" time="0.156" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[with_less_fee]" time="0.157" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_below_threshold]" time="0.153" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_at_threshold]" time="0.261" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_above_threshold]" time="0.265" /><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-010-banned-entity-not-allowed-banned-sender]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-010-banned-entity-not-allowed-banned-paymaster]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-010-banned-entity-not-allowed-banned-factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-020-throttled-entity-allowed-limited-count-throttled-sender]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-020-throttled-entity-allowed-limited-count-throttled-paymaster]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-020-throttled-entity-allowed-limited-count-throttled-factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-sender]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-paymaster]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_max_allowed_ops_unstaked_sender" time="0.763" /><testcase classname="tests.single.bundle.test_bundle" name="test_max_allowed_ops_staked_sender" time="0.882" /><testcase classname="tests.single.bundle.test_bundle" name="test_ban_user_op_access_other_ops_sender_in_bundle" time="0.776" /><testcase classname="tests.single.bundle.test_bundle" name="test_ban_user_sender_double_role_in_bundle" time="0.392" /><testcase classname="tests.single.bundle.test_bundle" name="test_stake_check_in_bundler" time="0.528" /><testcase classname="tests.single.bundle.test_codehash" name="test_codehash_changed" time="0.221"><skipped type="pytest.skip" message="no self destruct. can't check code change..">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_codehash.py:54: no self destruct. can't check code change..</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_paymaster" name="test_paymaster_deposit" time="0.772"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x07cbf54f989f59768d38b4374fe107c03baf2ba55aa19ceb7fc641af162896be', id=98)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_paymaster.py:48: in test_paymaster_deposit
    assert_rpc_error(res, "too low", RPCErrorCode.PAYMASTER_DEPOSIT_TOO_LOW)
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x07cbf54f989f59768d38b4374fe107c03baf2ba55aa19ceb7fc641af162896be', id=98)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok-]" time="0.360" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok-CALL:&gt;]" time="0.581" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok-DELEGATECALL:&gt;]" time="0.672" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_read][drop-]" time="0.377" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_read][drop-CALL:&gt;]" time="0.608" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_read][drop-DELEGATECALL:&gt;]" time="0.683" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_write][drop-]" time="0.393" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_write][drop-CALL:&gt;]" time="0.613" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_write][drop-DELEGATECALL:&gt;]" time="0.696" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tstore][drop-]" time="0.514"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tstore][drop-CALL:&gt;]" time="0.758"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tstore][drop-DELEGATECALL:&gt;]" time="0.852"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tload][drop-]" time="0.609"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tload][drop-CALL:&gt;]" time="0.751"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tload][drop-DELEGATECALL:&gt;]" time="0.848"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage][drop-]" time="0.530" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage][drop-CALL:&gt;]" time="0.788" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage][drop-DELEGATECALL:&gt;]" time="0.882" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage_struct][drop-]" time="0.592" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage_struct][drop-CALL:&gt;]" time="0.866" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage_struct][drop-DELEGATECALL:&gt;]" time="0.949" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok-]" time="0.427" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok-CALL:&gt;]" time="0.654" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok-DELEGATECALL:&gt;]" time="0.839" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-010)]unstaked][paymaster][account_transient_storage][ok-]" time="0.534" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-010)]unstaked][paymaster][account_transient_storage][ok-CALL:&gt;]" time="0.793" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-010)]unstaked][paymaster][account_transient_storage][ok-DELEGATECALL:&gt;]" time="0.877" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok-]" time="0.524" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok-CALL:&gt;]" time="0.770" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.859" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok-]" time="0.580" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok-CALL:&gt;]" time="0.870" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.937" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop-]" time="1.120" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop-CALL:&gt;]" time="1.000" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop-DELEGATECALL:&gt;]" time="1.083" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]unstaked][paymaster][context][drop-]" time="0.334" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]unstaked][paymaster][context][drop-CALL:&gt;]" time="0.362" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]unstaked][paymaster][context][drop-DELEGATECALL:&gt;]" time="0.290" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop-]" time="0.479" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop-CALL:&gt;]" time="0.822" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop-DELEGATECALL:&gt;]" time="0.926" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop-]" time="0.575" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop-CALL:&gt;]" time="0.697" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.881" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop-]" time="0.460" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop-CALL:&gt;]" time="0.787" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.776" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok-]" time="0.383" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok-CALL:&gt;]" time="0.701" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok-DELEGATECALL:&gt;]" time="0.713" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_read][ok-]" time="0.403" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_read][ok-CALL:&gt;]" time="0.726" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_read][ok-DELEGATECALL:&gt;]" time="0.819" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_write][ok-]" time="0.414" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_write][ok-CALL:&gt;]" time="0.742" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_write][ok-DELEGATECALL:&gt;]" time="0.828" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage][ok-]" time="0.652" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage][ok-CALL:&gt;]" time="0.917" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage][ok-DELEGATECALL:&gt;]" time="0.996" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage_struct][ok-]" time="0.711" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage_struct][ok-CALL:&gt;]" time="0.994" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="1.082" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][paymaster_reference_storage_struct][ok-]" time="0.710" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][paymaster_reference_storage_struct][ok-CALL:&gt;]" time="0.898" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][paymaster_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="1.078" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok-]" time="0.545" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok-CALL:&gt;]" time="0.677" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok-DELEGATECALL:&gt;]" time="0.886" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok-]" time="0.646" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok-CALL:&gt;]" time="0.790" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.877" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok-]" time="0.686" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok-CALL:&gt;]" time="0.980" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="1.056" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok-]" time="0.979" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok-CALL:&gt;]" time="1.025" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok-DELEGATECALL:&gt;]" time="1.092" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]staked][paymaster][context][ok-]" time="0.357" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]staked][paymaster][context][ok-CALL:&gt;]" time="0.280" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]staked][paymaster][context][ok-DELEGATECALL:&gt;]" time="0.313" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop-]" time="0.612" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop-CALL:&gt;]" time="0.857" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop-DELEGATECALL:&gt;]" time="0.846" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok-]" time="0.601" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok-CALL:&gt;]" time="0.845" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok-DELEGATECALL:&gt;]" time="0.931" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop-]" time="0.594" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop-CALL:&gt;]" time="0.807" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.894" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop-]" time="0.583" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop-CALL:&gt;]" time="0.708" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.894" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok-]" time="0.410" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok-CALL:&gt;]" time="0.576" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok-DELEGATECALL:&gt;]" time="0.597" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_read][drop-]" time="0.512" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_read][drop-CALL:&gt;]" time="0.490" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_read][drop-DELEGATECALL:&gt;]" time="0.535" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_write][drop-]" time="0.421" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_write][drop-CALL:&gt;]" time="0.487" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_write][drop-DELEGATECALL:&gt;]" time="0.505" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][factory_reference_storage][drop-]" time="0.334" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][factory_reference_storage][drop-CALL:&gt;]" time="0.638" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][factory_reference_storage][drop-DELEGATECALL:&gt;]" time="0.698" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][factory_reference_storage_struct][drop-]" time="0.338" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][factory_reference_storage_struct][drop-CALL:&gt;]" time="0.593" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][factory_reference_storage_struct][drop-DELEGATECALL:&gt;]" time="0.742" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok-]" time="0.316" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok-CALL:&gt;]" time="0.595" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok-DELEGATECALL:&gt;]" time="0.615" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop-]" time="0.423" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop-CALL:&gt;]" time="0.638" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop-DELEGATECALL:&gt;]" time="0.691" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop-]" time="0.435" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop-CALL:&gt;]" time="0.577" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop-DELEGATECALL:&gt;]" time="0.632" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop-]" time="0.335" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop-CALL:&gt;]" time="0.630" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop-DELEGATECALL:&gt;]" time="0.671" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok-]" time="0.444" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok-CALL:&gt;]" time="0.571" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok-DELEGATECALL:&gt;]" time="0.615" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop-]" time="0.393" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop-CALL:&gt;]" time="0.641" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.700" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop-]" time="0.396" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop-CALL:&gt;]" time="0.645" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.707" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop-]" time="0.396" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop-CALL:&gt;]" time="0.644" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.603" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop-]" time="0.381" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop-CALL:&gt;]" time="0.591" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.636" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop-]" time="0.392" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop-CALL:&gt;]" time="0.643" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.676" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop-]" time="0.397" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop-CALL:&gt;]" time="0.645" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.707" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop-]" time="0.296" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop-CALL:&gt;]" time="0.633" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.589" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop-]" time="0.523" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop-CALL:&gt;]" time="0.693" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.709" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop-]" time="0.453" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop-CALL:&gt;]" time="0.629" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.551" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok-]" time="0.433" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok-CALL:&gt;]" time="0.492" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok-DELEGATECALL:&gt;]" time="0.615" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_read][ok-]" time="0.438" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_read][ok-CALL:&gt;]" time="0.609" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_read][ok-DELEGATECALL:&gt;]" time="0.626" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_write][ok-]" time="0.442" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_write][ok-CALL:&gt;]" time="0.610" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_write][ok-DELEGATECALL:&gt;]" time="0.630" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage][ok-]" time="0.461" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage][ok-CALL:&gt;]" time="0.668" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage][ok-DELEGATECALL:&gt;]" time="0.727" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage_struct][ok-]" time="0.471" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage_struct][ok-CALL:&gt;]" time="0.717" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.773" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok-]" time="0.444" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok-CALL:&gt;]" time="0.619" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok-DELEGATECALL:&gt;]" time="0.654" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok-]" time="0.450" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok-CALL:&gt;]" time="0.554" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.726" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok-]" time="0.456" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok-CALL:&gt;]" time="0.703" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.787" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop-]" time="0.464" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop-CALL:&gt;]" time="0.676" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop-DELEGATECALL:&gt;]" time="0.701" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok-]" time="0.358" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok-CALL:&gt;]" time="0.545" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok-DELEGATECALL:&gt;]" time="0.693" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop-]" time="0.548" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop-CALL:&gt;]" time="0.706" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.830" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop-]" time="0.470" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop-CALL:&gt;]" time="0.549" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.669" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok-]" time="0.744" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok-CALL:&gt;]" time="0.565" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok-DELEGATECALL:&gt;]" time="0.558" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok-]" time="0.396" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok-CALL:&gt;]" time="0.584" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok-DELEGATECALL:&gt;]" time="0.638" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok-]" time="0.478" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok-CALL:&gt;]" time="0.723" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.615" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop-]" time="0.498" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop-CALL:&gt;]" time="0.423" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop-DELEGATECALL:&gt;]" time="0.400" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok-]" time="1.015"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=496)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=496)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok-CALL:&gt;]" time="0.756"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=498)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=498)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------
CALL:&gt; funding test_rules_target 0x7F454bfB9f2f7Ce40f8056830e3367ED2b5010Dc

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok-DELEGATECALL:&gt;]" time="0.753"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=500)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=500)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][ok-]" time="0.863"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=502)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=502)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][ok-CALL:&gt;]" time="1.147"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=504)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=504)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][ok-DELEGATECALL:&gt;]" time="1.235"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=506)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=506)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok-]" time="0.395" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok-CALL:&gt;]" time="0.632" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.631" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop-]" time="0.304" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop-CALL:&gt;]" time="0.625" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop-DELEGATECALL:&gt;]" time="0.615" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop-]" time="0.397" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop-CALL:&gt;]" time="0.684" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.578" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop-]" time="0.345" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop-CALL:&gt;]" time="0.620" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.622" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok-]" time="0.402" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok-CALL:&gt;]" time="0.590" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok-DELEGATECALL:&gt;]" time="0.586" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok-]" time="0.417" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok-CALL:&gt;]" time="0.614" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok-DELEGATECALL:&gt;]" time="0.605" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok-]" time="0.417" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok-CALL:&gt;]" time="0.639" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.636" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok-]" time="0.423" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok-CALL:&gt;]" time="0.660" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.657" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop-]" time="0.520" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop-CALL:&gt;]" time="0.699" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.700" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop-]" time="0.341" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop-CALL:&gt;]" time="0.647" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.644" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop-]" time="0.431" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop-CALL:&gt;]" time="0.677" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop-DELEGATECALL:&gt;]" time="0.651" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok-]" time="0.433" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok-CALL:&gt;]" time="0.643" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok-DELEGATECALL:&gt;]" time="0.640" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop-]" time="0.321" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop-CALL:&gt;]" time="0.644" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop-DELEGATECALL:&gt;]" time="0.636" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop-]" time="0.308" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop-CALL:&gt;]" time="0.641" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop-DELEGATECALL:&gt;]" time="0.539" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok-]" time="0.432" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok-CALL:&gt;]" time="0.661" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok-DELEGATECALL:&gt;]" time="0.664" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok-]" time="0.432" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok-CALL:&gt;]" time="0.692" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok-DELEGATECALL:&gt;]" time="0.583" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop-]" time="0.260" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop-CALL:&gt;]" time="0.605" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.595" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop-]" time="0.371" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop-CALL:&gt;]" time="0.501" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.600" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop-]" time="0.266" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop-CALL:&gt;]" time="0.604" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.600" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop-]" time="0.376" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop-CALL:&gt;]" time="0.590" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop-DELEGATECALL:&gt;]" time="0.599" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop-]" time="0.379" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop-CALL:&gt;]" time="0.598" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop-DELEGATECALL:&gt;]" time="0.495" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop-]" time="0.412" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop-CALL:&gt;]" time="0.700" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop-DELEGATECALL:&gt;]" time="0.589" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop-]" time="0.361" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop-CALL:&gt;]" time="0.489" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.570" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop-]" time="0.361" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop-CALL:&gt;]" time="0.591" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.585" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop-]" time="0.364" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop-CALL:&gt;]" time="0.612" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.497" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop-]" time="0.360" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop-CALL:&gt;]" time="0.597" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.600" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok-]" time="0.380" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok-CALL:&gt;]" time="0.644" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok-DELEGATECALL:&gt;]" time="0.536" /><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_eip_7702_tx" time="0.072"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0x90ec…92d4 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=670)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:36: in test_send_eip_7702_tx
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0x90ec…92d4 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=670)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_eip_7702_tx_with_initcode" time="0.070"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=672)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:85: in test_send_eip_7702_tx_with_initcode
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=672)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_eip_7702_tx_with_flag_no_initcode" time="0.072"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=674)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:127: in test_send_eip_7702_tx_with_flag_no_initcode
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=674)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_post_eip_7702_tx[0]" time="0.075"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0xd1e0…a96b is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=676)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:156: in test_send_post_eip_7702_tx
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0xd1e0…a96b is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=676)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_post_eip_7702_tx[1337]" time="0.073"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0x7a0b…defa is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=678)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:156: in test_send_post_eip_7702_tx
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0x7a0b…defa is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=678)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_bad_eip_7702_drop_userop" time="0.073"><failure message="AssertionError: assert -32602 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;&#10; +  where -32602 = Error(code=-32602, message='sender 0xde02…0d47 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=680).code">tests/single/eip7702/test_eip7702_tuple_userop.py:208: in test_send_bad_eip_7702_drop_userop
    assert_rpc_error(response, "", RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT)
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32602 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;
E    +  where -32602 = Error(code=-32602, message='sender 0xde02…0d47 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=680).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_nonsender_eip_7702_drop_userop" time="0.159"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x5d547e1c03296c69cb6f2c9a12e5daa33887a469a9663e980aad68d334160fde', id=682)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:223: in test_send_nonsender_eip_7702_drop_userop
    assert_rpc_error(userop.send(), "sender", RPCErrorCode.INVALID_FIELDS)
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x5d547e1c03296c69cb6f2c9a12e5daa33887a469a9663e980aad68d334160fde', id=682)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_wrongchain_eip_7702_drop_userop" time="0.073"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0x8822…078c is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=684)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:244: in test_send_wrongchain_eip_7702_drop_userop
    assert_ok(userop.send())
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0x8822…078c is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=684)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.gas.test_eip_7623" name="test_normal_calldata" time="0.157"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32500, message='AA26 over verificationGasLimit', data=None, id=686)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/gas/test_eip_7623.py:35: in test_normal_calldata
    assert_ok(op.send())
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32500, message='AA26 over verificationGasLimit', data=None, id=686)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.gas.test_eip_7623" name="test_huge_calldata_failed" time="0.152"><failure message="AssertionError: assert -32500 == &lt;RPCErrorCode.INVALID_FIELDS: -32602&gt;&#10; +  where -32500 = Error(code=-32500, message='AA26 over verificationGasLimit', data=None, id=688).code">tests/single/gas/test_eip_7623.py:55: in test_huge_calldata_failed
    assert_rpc_error(ret, "preVerificationGas", RPCErrorCode.INVALID_FIELDS)
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32500 == &lt;RPCErrorCode.INVALID_FIELDS: -32602&gt;
E    +  where -32500 = Error(code=-32500, message='AA26 over verificationGasLimit', data=None, id=688).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.gas.test_eip_7623" name="test_huge_calldata_with_callgas" time="0.179"><failure message="AssertionError: assert -32500 == &lt;RPCErrorCode.INVALID_FIELDS: -32602&gt;&#10; +  where -32500 = Error(code=-32500, message='AA26 over verificationGasLimit', data=None, id=690).code">tests/single/gas/test_eip_7623.py:74: in test_huge_calldata_with_callgas
    assert_rpc_error(ret, "preVerificationGas", RPCErrorCode.INVALID_FIELDS)
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32500 == &lt;RPCErrorCode.INVALID_FIELDS: -32602&gt;
E    +  where -32500 = Error(code=-32500, message='AA26 over verificationGasLimit', data=None, id=690).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.gas.test_eip_7623" name="test_estimate_huge_calldata" time="10.034"><failure message="requests.exceptions.ReadTimeout: HTTPConnectionPool(host='localhost', port=3000): Read timed out. (read timeout=10)">.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:536: in _make_request
    response = conn.getresponse()
.venv/lib/python3.10/site-packages/urllib3/connection.py:464: in getresponse
    httplib_response = super().getresponse()
../../../../.local/share/pdm/python/cpython@3.10.17/lib/python3.10/http/client.py:1375: in getresponse
    response.begin()
../../../../.local/share/pdm/python/cpython@3.10.17/lib/python3.10/http/client.py:318: in begin
    version, status, reason = self._read_status()
../../../../.local/share/pdm/python/cpython@3.10.17/lib/python3.10/http/client.py:279: in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
../../../../.local/share/pdm/python/cpython@3.10.17/lib/python3.10/socket.py:717: in readinto
    return self._sock.recv_into(b)
E   TimeoutError: timed out

The above exception was the direct cause of the following exception:
.venv/lib/python3.10/site-packages/requests/adapters.py:667: in send
    resp = conn.urlopen(
.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:843: in urlopen
    retries = retries.increment(
.venv/lib/python3.10/site-packages/urllib3/util/retry.py:474: in increment
    raise reraise(type(error), error, _stacktrace)
.venv/lib/python3.10/site-packages/urllib3/util/util.py:39: in reraise
    raise value
.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:789: in urlopen
    response = self._make_request(
.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:538: in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:369: in _raise_timeout
    raise ReadTimeoutError(
E   urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(host='localhost', port=3000): Read timed out. (read timeout=10)

During handling of the above exception, another exception occurred:
tests/single/gas/test_eip_7623.py:94: in test_estimate_huge_calldata
    ).send()
tests/types.py:60: in send
    postres = requests.post(url, json=asdict(self), timeout=10).json()
.venv/lib/python3.10/site-packages/requests/api.py:115: in post
    return request("post", url, data=data, json=json, **kwargs)
.venv/lib/python3.10/site-packages/requests/api.py:59: in request
    return session.request(method=method, url=url, **kwargs)
.venv/lib/python3.10/site-packages/requests/sessions.py:589: in request
    resp = self.send(prep, **send_kwargs)
.venv/lib/python3.10/site-packages/requests/sessions.py:703: in send
    r = adapter.send(request, **kwargs)
.venv/lib/python3.10/site-packages/requests/adapters.py:713: in send
    raise ReadTimeout(e, request=request)
E   requests.exceptions.ReadTimeout: HTTPConnectionPool(host='localhost', port=3000): Read timed out. (read timeout=10)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_no_factory[CREATE]" time="0.347" /><testcase classname="tests.single.opbanning.test_create" name="test_account_no_factory[CREATE2]" time="0.356" /><testcase classname="tests.single.opbanning.test_create" name="test_paymaster_banned_create[CREATE]" time="0.366" /><testcase classname="tests.single.opbanning.test_create" name="test_paymaster_banned_create[CREATE2]" time="0.341" /><testcase classname="tests.single.opbanning.test_create" name="test_account_create2_once" time="0.412" /><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=unstaked,op=CREATE,expect=ok,rule=OP-032]" time="0.314"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=704)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_create.py:136: in test_account_create_with_factory
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=704)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=unstaked,op=CREATE2,expect=err,rule=OP-032]" time="0.509"><failure message="AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'&#10; +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f77daf437b0&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f77daf437b0&gt; = 'account uses banned opcode: CREATE2'.lower&#10; +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f77daf40750&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f77daf40750&gt; = 'factory uses banned opcode: CREATE'.lower&#10; +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=706).message">tests/single/opbanning/test_create.py:138: in test_account_create_with_factory
    assert_rpc_error(
tests/utils.py:207: in assert_rpc_error
    assert message.lower() in response.message.lower()
E   AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'
E    +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f77daf437b0&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f77daf437b0&gt; = 'account uses banned opcode: CREATE2'.lower
E    +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f77daf40750&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f77daf40750&gt; = 'factory uses banned opcode: CREATE'.lower
E    +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=706).message</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=CREATE,expect=ok,rule=EREP-060]" time="0.449"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=708)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_create.py:136: in test_account_create_with_factory
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=708)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=nested-CREATE,expect=err,rule=EREP-060]" time="0.337"><failure message="AssertionError: assert 'account uses banned opcode: create' in 'factory uses banned opcode: create'&#10; +  where 'account uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f77dae1fc30&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f77dae1fc30&gt; = 'account uses banned opcode: CREATE'.lower&#10; +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f77daf437b0&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f77daf437b0&gt; = 'factory uses banned opcode: CREATE'.lower&#10; +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=710).message">tests/single/opbanning/test_create.py:138: in test_account_create_with_factory
    assert_rpc_error(
tests/utils.py:207: in assert_rpc_error
    assert message.lower() in response.message.lower()
E   AssertionError: assert 'account uses banned opcode: create' in 'factory uses banned opcode: create'
E    +  where 'account uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f77dae1fc30&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f77dae1fc30&gt; = 'account uses banned opcode: CREATE'.lower
E    +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f77daf437b0&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f77daf437b0&gt; = 'factory uses banned opcode: CREATE'.lower
E    +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=710).message</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=nested-CREATE2,expect=err,rule=EREP-060]" time="0.342"><failure message="AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'&#10; +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f77db148090&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f77db148090&gt; = 'account uses banned opcode: CREATE2'.lower&#10; +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f77db0b1590&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f77db0b1590&gt; = 'factory uses banned opcode: CREATE'.lower&#10; +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=712).message">tests/single/opbanning/test_create.py:138: in test_account_create_with_factory
    assert_rpc_error(
tests/utils.py:207: in assert_rpc_error
    assert message.lower() in response.message.lower()
E   AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'
E    +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f77db148090&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f77db148090&gt; = 'account uses banned opcode: CREATE2'.lower
E    +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f77db0b1590&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f77db0b1590&gt; = 'factory uses banned opcode: CREATE'.lower
E    +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=712).message</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=CREATE2,expect=ok,rule=EREP-060]" time="0.435"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=714)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_create.py:136: in test_account_create_with_factory
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=714)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS-]" time="0.230" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS-CALL:&gt;]" time="0.472" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS-DELEGATECALL:&gt;]" time="0.515" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER-]" time="0.370" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER-CALL:&gt;]" time="0.506" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER-DELEGATECALL:&gt;]" time="0.449" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP-]" time="0.328" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP-CALL:&gt;]" time="0.408" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP-DELEGATECALL:&gt;]" time="0.612" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE-]" time="0.333" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE-CALL:&gt;]" time="0.514" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE-DELEGATECALL:&gt;]" time="0.569" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY-]" time="0.238" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY-CALL:&gt;]" time="0.511" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY-DELEGATECALL:&gt;]" time="0.444" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE-]" time="0.330" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE-CALL:&gt;]" time="0.486" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE-DELEGATECALL:&gt;]" time="0.510" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT-]" time="0.237" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT-CALL:&gt;]" time="0.513" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT-DELEGATECALL:&gt;]" time="0.402" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE-]" time="0.221" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE-CALL:&gt;]" time="0.481" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE-DELEGATECALL:&gt;]" time="0.505" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE-]" time="0.240" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE-CALL:&gt;]" time="0.506" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE-DELEGATECALL:&gt;]" time="0.528" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE-]" time="0.215" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE-CALL:&gt;]" time="0.377" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE-DELEGATECALL:&gt;]" time="0.500" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN-]" time="0.319" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN-CALL:&gt;]" time="0.468" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN-DELEGATECALL:&gt;]" time="0.495" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH-]" time="0.324" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH-CALL:&gt;]" time="0.486" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH-DELEGATECALL:&gt;]" time="0.509" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE-]" time="0.337" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE-CALL:&gt;]" time="0.493" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE-DELEGATECALL:&gt;]" time="0.518" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2-]" time="0.338" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2-CALL:&gt;]" time="0.394" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2-DELEGATECALL:&gt;]" time="0.524" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT-]" time="0.239" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT-CALL:&gt;]" time="0.412" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT-DELEGATECALL:&gt;]" time="0.537" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBHASH-]" time="0.384"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xe5d3bd673d1338d7d17b85fa8df840f249c372c3a258ed75f2e9bfd8e4f395a4', id=806)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xe5d3bd673d1338d7d17b85fa8df840f249c372c3a258ed75f2e9bfd8e4f395a4', id=806)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBHASH-CALL:&gt;]" time="0.502"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x6c02fb427be714839660c59e7878798928c911dcf9e8cccd6d912f0f0736ca9c', id=808)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x6c02fb427be714839660c59e7878798928c911dcf9e8cccd6d912f0f0736ca9c', id=808)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBHASH-DELEGATECALL:&gt;]" time="0.517"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x10f5c119c758e6c411200f6873b9a70e952fa32fba52a73785c4705aa45ad4c2', id=810)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x10f5c119c758e6c411200f6873b9a70e952fa32fba52a73785c4705aa45ad4c2', id=810)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBBASEFEE-]" time="0.239"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x70b9ecf3a143718a3ea274a6f15e5da9bae5c9eaf3ee39d69de4ab55dd8380ee', id=812)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x70b9ecf3a143718a3ea274a6f15e5da9bae5c9eaf3ee39d69de4ab55dd8380ee', id=812)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBBASEFEE-CALL:&gt;]" time="0.410"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x45403f9ab19573251f5e095471b26cd549dc5106c0b71165c9489c0beb0d0d95', id=814)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x45403f9ab19573251f5e095471b26cd549dc5106c0b71165c9489c0beb0d0d95', id=814)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBBASEFEE-DELEGATECALL:&gt;]" time="0.526"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x1cb5f9bef52051be87dc1924714f1019d514734cf93655958a1371dd6c4de4a8', id=816)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x1cb5f9bef52051be87dc1924714f1019d514734cf93655958a1371dd6c4de4a8', id=816)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[BALANCE-]" time="0.351"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=818)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=818)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[BALANCE-CALL:&gt;]" time="0.404"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=820)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=820)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[BALANCE-DELEGATECALL:&gt;]" time="0.420"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=822)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=822)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[SELFBALANCE-]" time="0.367"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=824)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=824)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[SELFBALANCE-CALL:&gt;]" time="0.438"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=826)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=826)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[SELFBALANCE-DELEGATECALL:&gt;]" time="0.556"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=828)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=828)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL-]" time="0.369" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL-CALL:&gt;]" time="0.524" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL-DELEGATECALL:&gt;]" time="0.539" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL-]" time="0.369" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL-CALL:&gt;]" time="0.573" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL-DELEGATECALL:&gt;]" time="0.567" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS-]" time="0.265" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS-CALL:&gt;]" time="0.540" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS-DELEGATECALL:&gt;]" time="0.631" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER-]" time="0.370" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER-CALL:&gt;]" time="0.576" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER-DELEGATECALL:&gt;]" time="0.669" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP-]" time="0.395" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP-CALL:&gt;]" time="0.615" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP-DELEGATECALL:&gt;]" time="0.591" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE-]" time="0.495" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE-CALL:&gt;]" time="0.598" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE-DELEGATECALL:&gt;]" time="0.686" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY-]" time="0.405" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY-CALL:&gt;]" time="0.632" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY-DELEGATECALL:&gt;]" time="0.715" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE-]" time="0.383" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE-CALL:&gt;]" time="0.596" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE-DELEGATECALL:&gt;]" time="0.690" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT-]" time="0.412" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT-CALL:&gt;]" time="0.604" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT-DELEGATECALL:&gt;]" time="0.694" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE-]" time="0.393" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE-CALL:&gt;]" time="0.609" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE-DELEGATECALL:&gt;]" time="0.693" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE-]" time="0.444" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE-CALL:&gt;]" time="0.664" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE-DELEGATECALL:&gt;]" time="0.781" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE-]" time="0.389" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE-CALL:&gt;]" time="0.591" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE-DELEGATECALL:&gt;]" time="0.682" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN-]" time="0.380" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN-CALL:&gt;]" time="0.581" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN-DELEGATECALL:&gt;]" time="0.683" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH-]" time="0.410" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH-CALL:&gt;]" time="0.631" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH-DELEGATECALL:&gt;]" time="0.708" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE-]" time="0.501" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE-CALL:&gt;]" time="0.601" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE-DELEGATECALL:&gt;]" time="0.692" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2-]" time="0.409" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2-CALL:&gt;]" time="0.614" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2-DELEGATECALL:&gt;]" time="0.805" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT-]" time="0.451" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT-CALL:&gt;]" time="0.685" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT-DELEGATECALL:&gt;]" time="0.761" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBHASH-]" time="0.411"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x298e987529f4bd8b9368c417c41312f2efe20d4268634073aa1f1fe3e15fb749', id=932)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x298e987529f4bd8b9368c417c41312f2efe20d4268634073aa1f1fe3e15fb749', id=932)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBHASH-CALL:&gt;]" time="0.626"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xd21b80f41a42be4e003f20d9cdad4aef7c8f9bf93526058ecac418c7520f4280', id=934)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xd21b80f41a42be4e003f20d9cdad4aef7c8f9bf93526058ecac418c7520f4280', id=934)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBHASH-DELEGATECALL:&gt;]" time="0.816"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x43b5c1fd4e6b949ea441f524b01442d248045b4723743208dd4cef8da2ca8132', id=936)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x43b5c1fd4e6b949ea441f524b01442d248045b4723743208dd4cef8da2ca8132', id=936)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBBASEFEE-]" time="0.489"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xa38bad6141c657b3fec7fb5c740a06c4efe12738024cdce059d35c81a53c4c0b', id=938)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xa38bad6141c657b3fec7fb5c740a06c4efe12738024cdce059d35c81a53c4c0b', id=938)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBBASEFEE-CALL:&gt;]" time="0.666"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x4f941a443b11e4659b189565368bd64ed61d3da11ae533cfde65fc3a2da61574', id=940)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x4f941a443b11e4659b189565368bd64ed61d3da11ae533cfde65fc3a2da61574', id=940)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBBASEFEE-DELEGATECALL:&gt;]" time="0.749"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xb7491ca9aec65d96333c88661b33237ca8946a0158423794c307338acfade272', id=942)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xb7491ca9aec65d96333c88661b33237ca8946a0158423794c307338acfade272', id=942)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_allowed_opcode_when_staked[BALANCE]" time="0.411"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='paymaster uses banned opcode: BALANCE', data=None, id=944)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:113: in test_paymaster_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='paymaster uses banned opcode: BALANCE', data=None, id=944)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_allowed_opcode_when_staked[SELFBALANCE]" time="0.466"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='paymaster uses banned opcode: SELFBALANCE', data=None, id=946)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:113: in test_paymaster_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='paymaster uses banned opcode: SELFBALANCE', data=None, id=946)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GAS]" time="0.356" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[NUMBER]" time="0.258" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[TIMESTAMP]" time="0.360" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[COINBASE]" time="0.362" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[DIFFICULTY]" time="0.361" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BASEFEE]" time="0.370" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GASLIMIT]" time="0.365" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GASPRICE]" time="0.366" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[SELFBALANCE]" time="0.296" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BALANCE]" time="0.373" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[ORIGIN]" time="0.371" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOCKHASH]" time="0.373" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[CREATE]" time="0.383" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[CREATE2]" time="0.383" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[SELFDESTRUCT]" time="0.390" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOBHASH]" time="0.387"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x242a07c6e9ba7dd55a332476dd3e61d5ca6d46841a6f4fce710646497191b725', id=978)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:127: in test_factory_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x242a07c6e9ba7dd55a332476dd3e61d5ca6d46841a6f4fce710646497191b725', id=978)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOBBASEFEE]" time="0.431"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x1b7a75c05b1a0248484ed1fc3b123abc3ec9e27f12d670c21419441b222ba5a5', id=980)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:127: in test_factory_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x1b7a75c05b1a0248484ed1fc3b123abc3ec9e27f12d670c21419441b222ba5a5', id=980)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_allowed_opcode_when_staked[BALANCE]" time="0.398"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: BALANCE', data=None, id=982)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:153: in test_factory_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: BALANCE', data=None, id=982)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_allowed_opcode_when_staked[SELFBALANCE]" time="0.416"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: SELFBALANCE', data=None, id=984)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:153: in test_factory_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: SELFBALANCE', data=None, id=984)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_paymaster_on_account_failure" time="0.251"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/reputation/test_erep.py:71: in test_paymaster_on_account_failure
    set_reputation(paymaster.address, ops_seen=5, ops_included=2)
tests/utils.py:321: in set_reputation
    assert res.result
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_staked_factory_on_account_failure" time="0.848"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32603, message='gRPC error: Send bundle now with error: (code: 3, message: execution reverted, data: Some(String(&quot;0x65c8fd4d0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000000d4141323320726576657274656400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006408c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000114e6f2062756e646c6520666f7220796f7500000000000000000000000000000000000000000000000000000000000000000000000000000000000000&quot;)))\n\nLocation:\n    /app/crates/bundler/src/bundler.rs:123:29', data=None, id=994)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/reputation/test_erep.py:119: in test_staked_factory_on_account_failure
    send_bundle_now()
tests/utils.py:230: in send_bundle_now
    assert_ok(RPCRequest(method="debug_bundler_sendBundleNow").send(url))
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32603, message='gRPC error: Send bundle now with error: (code: 3, message: execution reverted, data: Some(String("0x65c8fd4d0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000000d4141323320726576657274656400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006408c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000114e6f2062756e646c6520666f7220796f7500000000000000000000000000000000000000000000000000000000000000000000000000000000000000")))\n\nLocation:\n    /app/crates/bundler/src/bundler.rs:123:29', data=None, id=994)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_account_on_entity_failure[staked]" time="0.482"><failure message="AssertionError: staked account should be blamed instead of paymaster&#10;assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]&#10;  &#10;  At index 1 diff: Reputation(address='0x91f8d12edc29bf52fcfef8a4579aad5703b8aca0', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0x91f8d12edc29bf52fcfef8a4579aad5703b8aca0', opsSeen=0, opsIncluded=0, status=0)&#10;  &#10;  Full diff:&#10;    [&#10;        Reputation(&#10;            address='0x2377d4f0abc658cec8c227137368c9bbaeea8d07',&#10;            opsSeen=1,&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;        Reputation(&#10;            address='0x91f8d12edc29bf52fcfef8a4579aad5703b8aca0',&#10;  -         opsSeen=0,&#10;  ?                 ^&#10;  +         opsSeen=1,&#10;  ?                 ^&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;    ]">tests/single/reputation/test_erep.py:174: in test_account_on_entity_failure
    assert get_reputations([sender.address, paymaster.address]) == [
E   AssertionError: staked account should be blamed instead of paymaster
E   assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]
E     
E     At index 1 diff: Reputation(address='0x91f8d12edc29bf52fcfef8a4579aad5703b8aca0', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0x91f8d12edc29bf52fcfef8a4579aad5703b8aca0', opsSeen=0, opsIncluded=0, status=0)
E     
E     Full diff:
E       [
E           Reputation(
E               address='0x2377d4f0abc658cec8c227137368c9bbaeea8d07',
E               opsSeen=1,
E               opsIncluded=0,
E               status=0,
E           ),
E           Reputation(
E               address='0x91f8d12edc29bf52fcfef8a4579aad5703b8aca0',
E     -         opsSeen=0,
E     ?                 ^
E     +         opsSeen=1,
E     ?                 ^
E               opsIncluded=0,
E               status=0,
E           ),
E       ]</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------
staking account &lt;web3._utils.datatypes.Contract object at 0x7f77db7e3340&gt;

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_account_on_entity_failure[unstaked]" time="0.268"><failure message="AssertionError: valid userop. staked paymaster should have opsSeen increment&#10;assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]&#10;  &#10;  At index 0 diff: Reputation(address='0x51dba99ba29087f27287a80bde83fdaaaa1debbf', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0x51dba99ba29087f27287a80bde83fdaaaa1debbf', opsSeen=0, opsIncluded=0, status=0)&#10;  &#10;  Full diff:&#10;    [&#10;        Reputation(&#10;            address='0x51dba99ba29087f27287a80bde83fdaaaa1debbf',&#10;  -         opsSeen=0,&#10;  ?                 ^&#10;  +         opsSeen=1,&#10;  ?                 ^&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;        Reputation(&#10;            address='0xea0f21e04b87dfe6c4b142c105669fa132c8bb18',&#10;            opsSeen=1,&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;    ]">tests/single/reputation/test_erep.py:156: in test_account_on_entity_failure
    assert get_reputations([sender.address, paymaster.address]) == [
E   AssertionError: valid userop. staked paymaster should have opsSeen increment
E   assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]
E     
E     At index 0 diff: Reputation(address='0x51dba99ba29087f27287a80bde83fdaaaa1debbf', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0x51dba99ba29087f27287a80bde83fdaaaa1debbf', opsSeen=0, opsIncluded=0, status=0)
E     
E     Full diff:
E       [
E           Reputation(
E               address='0x51dba99ba29087f27287a80bde83fdaaaa1debbf',
E     -         opsSeen=0,
E     ?                 ^
E     +         opsSeen=1,
E     ?                 ^
E               opsIncluded=0,
E               status=0,
E           ),
E           Reputation(
E               address='0xea0f21e04b87dfe6c4b142c105669fa132c8bb18',
E               opsSeen=1,
E               opsIncluded=0,
E               status=0,
E           ),
E       ]</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------
unstaked account &lt;web3._utils.datatypes.Contract object at 0x7f77db175570&gt;

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_replace" name="test_replace_paymaster" time="0.564"><failure message="AssertionError: replace with unmodified userop&#10;assert 2 == 1&#10; +  where 2 = pm_opsSeen('0xeF2D15188fE73EcA98C0186D74E0F12028ca90cb')&#10; +    where '0xeF2D15188fE73EcA98C0186D74E0F12028ca90cb' = &lt;web3._utils.datatypes.Contract object at 0x7f77db05f4c0&gt;.address">tests/single/reputation/test_replace.py:37: in test_replace_paymaster
    assert pm_opsSeen(pm1.address) == 1, "replace with unmodified userop"
E   AssertionError: replace with unmodified userop
E   assert 2 == 1
E    +  where 2 = pm_opsSeen('0xeF2D15188fE73EcA98C0186D74E0F12028ca90cb')
E    +    where '0xeF2D15188fE73EcA98C0186D74E0F12028ca90cb' = &lt;web3._utils.datatypes.Contract object at 0x7f77db05f4c0&gt;.address</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_reputation" name="test_staked_entity_reputation_threshold[with_factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/reputation/test_reputation.py:61: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_reputation" name="test_staked_entity_reputation_threshold[without_factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/reputation/test_reputation.py:61: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_chainId" name="test_eth_chainId[]" time="0.008" /><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas[]" time="0.170" /><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas_execution_revert" time="0.178"><failure message="AttributeError: expected error object, got:&#10;Ok(result={'preVerificationGas': '0xb413', 'verificationGasLimit': '0xeffe', 'callGasLimit': '0x2f44'}, id=1022)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/rpc/test_eth_estimateUserOperationGas.py:32: in test_eth_estimateUserOperationGas_execution_revert
    assert_rpc_error(response, "", RPCErrorCode.EXECUTION_REVERTED)
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result={'preVerificationGas': '0xb413', 'verificationGasLimit': '0xeffe', 'callGasLimit': '0x2f44'}, id=1022)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas_simulation_revert" time="0.028"><failure message="assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;&#10; +  where -32603 = Error(code=-32603, message='other error: FailedOpWithRevert(FailedOpWithRevert { op_index: 0, reason: &quot;AA23 reverted&quot;,...0000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000) })', data=None, id=1024).code">tests/single/rpc/test_eth_estimateUserOperationGas.py:40: in test_eth_estimateUserOperationGas_simulation_revert
    assert_rpc_error(response, "dead signature", RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT)
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;
E    +  where -32603 = Error(code=-32603, message='other error: FailedOpWithRevert(FailedOpWithRevert { op_index: 0, reason: "AA23 reverted",...0000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000) })', data=None, id=1024).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_getUserOperationByHash" name="test_eth_getUserOperationByHash[]" time="0.354"><failure message="jsonschema.exceptions.ValidationError: None is not valid under any of the given schemas&#10;&#10;Failed validating 'oneOf' in schema['properties']['userOperation']['properties']['paymasterVerificationGasLimit']:&#10;    {'title': 'paymaster verification gas limit',&#10;     'oneOf': [{'type': 'string', 'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$'},&#10;               {'type': 'number'}],&#10;     'description': 'The amount of gas to allocate for the paymaster '&#10;                    'validation code'}&#10;&#10;On instance['userOperation']['paymasterVerificationGasLimit']:&#10;    None">tests/single/rpc/test_eth_getUserOperationByHash.py:24: in test_eth_getUserOperationByHash
    validate(instance=response.result, schema=schema)
.venv/lib/python3.10/site-packages/jsonschema/validators.py:1332: in validate
    raise error
E   jsonschema.exceptions.ValidationError: None is not valid under any of the given schemas
E   
E   Failed validating 'oneOf' in schema['properties']['userOperation']['properties']['paymasterVerificationGasLimit']:
E       {'title': 'paymaster verification gas limit',
E        'oneOf': [{'type': 'string', 'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$'},
E                  {'type': 'number'}],
E        'description': 'The amount of gas to allocate for the paymaster '
E                       'validation code'}
E   
E   On instance['userOperation']['paymasterVerificationGasLimit']:
E       None</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_getUserOperationByHash" name="test_eth_getUserOperationByHash_error" time="0.005" /><testcase classname="tests.single.rpc.test_eth_getUserOperationReceipt" name="test_eth_getUserOperationReceipt[]" time="0.369" /><testcase classname="tests.single.rpc.test_eth_getUserOperationReceipt" name="test_eth_getUserOperationReceipt_error" time="0.005" /><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation[]" time="0.334"><failure message="AssertionError: assert '0x479449e1b1...0bec6a64fc938' == '0xaff536244e...610b2347caa85'&#10;  &#10;  - 0xaff536244e526d83e7dd7fa56bfe1c3ac370f3040d9b612a3f2610b2347caa85&#10;  + 0x479449e1b18f71a12dc19836ccae89e541f4536b51aac12c03e0bec6a64fc938">tests/single/rpc/test_eth_sendUserOperation.py:19: in test_eth_sendUserOperation
    assert response.result == userop_hash(helper_contract, userop)
E   AssertionError: assert '0x479449e1b1...0bec6a64fc938' == '0xaff536244e...610b2347caa85'
E     
E     - 0xaff536244e526d83e7dd7fa56bfe1c3ac370f3040d9b612a3f2610b2347caa85
E     + 0x479449e1b18f71a12dc19836ccae89e541f4536b51aac12c03e0bec6a64fc938</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation_revert" time="0.043"><failure message="AssertionError: assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;&#10; +  where -32603 = Error(code=-32603, message='other error: 0, AA23 reverted, 0x08c379a00000000000000000000000000000000000000000000000000...00000000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000', data=None, id=1041).code">tests/single/rpc/test_eth_sendUserOperation.py:32: in test_eth_sendUserOperation_revert
    assert_rpc_error(
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;
E    +  where -32603 = Error(code=-32603, message='other error: 0, AA23 reverted, 0x08c379a00000000000000000000000000000000000000000000000000...00000000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000', data=None, id=1041).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation_invalid_signature" time="0.029"><failure message="AssertionError: assert -32503 == &lt;RPCErrorCode.INVALID_SIGNATURE: -32507&gt;&#10; +  where -32503 = Error(code=-32503, message='already expired', data=None, id=1044).code">tests/single/rpc/test_eth_sendUserOperation.py:39: in test_eth_sendUserOperation_invalid_signature
    assert_rpc_error(
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32503 == &lt;RPCErrorCode.INVALID_SIGNATURE: -32507&gt;
E    +  where -32503 = Error(code=-32503, message='already expired', data=None, id=1044).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_supportedEntryPoints" name="test_eth_supportedEntryPoints[]" time="0.009" /></testsuite></testsuites>